catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlResult.java

```diff
@@ -126,10 +145,26 @@ public CqlResult(
       countMatches(searchTerms, mc);
     }
 
-    actions = actionRegistry.list(result.getMetacard()).stream().collect(Collectors.toList());
+    actions =
+        actionRegistry
+            .list(result.getMetacard())
+            .stream()
+            .map(cqlResult -> new CqlAction(cqlResult, cqlResult.getId()))
```

Comment:
The object being mapped here is an `Action` so the parameter should be called `action`.